spec/index.html

```diff
@@ -11125,7 +11126,8 @@ <h3>Grammar</h3>
             nor in <code><a href="#rDeleteData">DELETE DATA</a></code>.</li>
           <li>Rules for limiting the use of blank node identifiers are given in <a href="#grammarBNodes">section 19.6</a>.</li>
           <li>The number of variables in the variable list of <code>VALUES</code> block 
-            must be the same as the number of each list of associated values in the <code>DataBlock</code>.</li>
```

Comment:
Suggestion (not blocking):

Keep the original text and have a new list item. Then,m each list item is about one requirement of VALUES.

```suggestion
          <li>The number of variables in the variable list of <code>VALUES</code> block 
            must be the same as the number of each list of associated values in the <code>DataBlock</code>.</li>
          <li>Variables in the variable list of <code>VALUES</code> block must be unique.</li>
```